toxcore/DHT.c

```diff
@@ -985,24 +985,24 @@ static unsigned int ping_node_from_getnodes_ok(DHT *dht, const uint8_t *public_k
     for (i = 0; i < dht->num_friends; ++i) {
         bool store_ok = 0;
 
-        DHT_Friend *friend = &dht->friends_list[i];
+        DHT_Friend *dht_friend = &dht->friends_list[i];
```

Comment:
Again, yes. Friend in the context of DHT is problematic.

Breaking merges or otherwise this is a positive change.